Meta/Lagom/Fuzzers/FuzzILBMLoader.cpp

```diff
@@ -0,0 +1,18 @@
+/*
```

Comment:
From [CONTRIBUTING](https://github.com/SerenityOS/serenity/blob/master/CONTRIBUTING.md#code-submission-policy):
> Write the commit message subject line in the imperative mood ("Foo: Change the way dates work", not "Foo: Changed the way dates work").

And this hasn't much to do with `LibGfx`, you can make two separate commits or otherwise use `Meta+Tests` for the category.

Reply:
I used `Meta+Tests` for the category.

Reply:
You still don't use the imperative mood for this commit.
"Meta+Tests: Add a fuzzer and a test for the ILBM decoder" would be better

Reply:
Done :)